website/src/main/tut/docs/users/installation.md

```diff
@@ -139,38 +139,75 @@ It is possible to use scalafix with scala-maven-plugin but it requires a custom
 
 ### Install semanticdb compiler plugin
 
-First, download the `semanticdb-scalac` compiler plugin which corresponds to your exact scala version of your project, down to the patch number.
+You can either install the semanticdb compiler plugin manually or let maven resolve the dependency for you.
 
-To begin with, it's recommended to install the coursier command line interface https://github.com/coursier/coursier.
+To install the plugin manually, first, download the `semanticdb-scalac` compiler plugin which corresponds to the exact scala version of your project(down to the patch number).
+
+It's recommended to do it via the [coursier command line interface](https://github.com/coursier/coursier) which simplifies downloading and launching library artifacts.
+
+To install coursier:
 
 ```
 wget https://github.com/coursier/coursier/raw/master/coursier && chmod +x coursier && ./coursier --help
 ```
 
-Coursier is a tool to download and launch library artifacts.
-Once you have coursier installed, assuming you are on {{ site.scala212 }}:
+Then, once you have coursier installed, assuming you are on {{ site.scala212 }}, download the plugin:
 
 ```
-coursier fetch --intransitive org.scalameta:semanticdb-scalac_{{ site.scala212 }}:{{ site.scalametaVersion }}
+coursier fetch --intransitive org.scalameta:semanticdb-scalac_{{ site.scala212 }}:2.1.7
 ```
 
-You can also use `wget` or a simalar tool to retrieve the jar from `https://repo1.maven.org/maven2/org/scalameta/semanticdb-scalac_{{ site.scala212 }}/{{ site.scalametaVersion }}/semanticdb-scalac_{{ site.scala212 }}-{{ site.scalametaVersion }}-javadoc.jar`.
+Alternatively, you can also use `wget` or a simalar tool to retrieve the jar from `https://repo1.maven.org/maven2/org/scalameta/semanticdb-scalac_{{ site.scala212 }}/2.1.7/semanticdb-scalac_{{ site.scala212 }}-2.1.7.jar`.
+
+If you prefer using maven profile instead of manual download, add the following snippet to your maven `pom.xml`:
+
+```
+<profile>
+  <id>scalafix</id>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>net.alchim31.maven</groupId>
+        <artifactId>scala-maven-plugin</artifactId>
+        <configuration>
+          <compilerPlugins>
+            <compilerPlugin>
+              <groupId>org.scalameta</groupId>
+              <artifactId>semanticdb-scalac_{{ site.scala212 }}</artifactId>
+              <version>2.1.7</version>
```

Comment:
Let's use the dynamic version here so it won't have to be updated manually every release